### PR TITLE
Show all venues

### PR DESCRIPTION
**Pull request by irais-valenzuela**
Can query yelp API for single and all venues with backend post route. Also, made a thunk in redux store that receives data from the post route in the backend.

### CODE REVIEW

File: client/components/AllVenues.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+const AllVenues = (props) => {
+  const [venueName, setVenueName] = useState('');
+  const [address, setAddress] = useState('');
+  const [price, setPrice] = useState('');
+  const [rating, setRating] = useState('');
+  const [hoursOfOperation, setHoursOfOperation] = useState('');
+
+  useEffect(() => {
+    console.log('in use effect');
```
Comment (j-yeh) [inline client/components/AllVenues.js:14]: Console log here

File: client/components/AllVenues.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+const mapStateToProps = (state) => {
+  return {
+    venues: state.venues,
+  };
+};
+
+const mapDispatch = (dispatch) => {
+  return {
+    getVenues: () => {
+      console.log('in map dispatch');
```
Comment (j-yeh) [inline client/components/AllVenues.js:39]: Console log here

File: client/redux/venues.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+
+const GET_VENUES = 'GET VENUES';
```
Comment (j-yeh) [inline client/redux/venues.js:3]: Does it need to be ‘GET_VENUES’ ?

File: client/redux/venues.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+const getVenuesCreator = (venues = {
+  type: GET_VENUES,
+  venues,
+});
+
+export const getVenuesThunk = () => async (dispatch) => {
+  try {
+    const {data} = await axios.post('/api/venues')
+    const businessArray = data.data.search.business
+    console.log("businessArray", businessArray)
```
Comment (j-yeh) [inline client/redux/venues.js:14]: Console log here

File: client/redux/venues.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    dispatch(getVenuesCreator(businessArray))
+  } catch(error) {
+    console.error(error)
+  }
+};
+
+export default function venues(state = [], action) {
+  console.log('action', action);
```
Comment (j-yeh) [inline client/redux/venues.js:22]: Console log here

File: client/redux/venues.js (hunk 3 above)
Comment (irais-valenzuela) [reply client/redux/venues.js:3]: Yes! Good catch!

**Review by j-yeh — APPROVED** [review]

**Pull request merged**